Common/src/gRPC/Services/GrpcResultsService.cs

```diff
@@ -0,0 +1,76 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2022. All rights reserved.
+//   W. Kirschenmann   <wkirschenmann@aneo.fr>
+//   J. Gurhem         <jgurhem@aneo.fr>
+//   D. Dubuc          <ddubuc@aneo.fr>
+//   L. Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F. Lemaitre       <flemaitre@aneo.fr>
+//   S. Djebbar        <sdjebbar@aneo.fr>
+//   J. Fonseca        <jfonseca@aneo.fr>
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Linq;
+using System.Threading.Tasks;
+
+using ArmoniK.Api.Common.Utils;
+using ArmoniK.Api.gRPC.V1.Results;
+using ArmoniK.Core.Common.Auth.Authentication;
+using ArmoniK.Core.Common.Storage;
+
+using Grpc.Core;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Logging;
+
+namespace ArmoniK.Core.Common.gRPC.Services;
+
+[Authorize(AuthenticationSchemes = Authenticator.SchemeName)]
+public class GrpcResultsService : Results.ResultsBase
+{
+  private readonly ILogger<GrpcResultsService> logger_;
+  private readonly IResultTable                resultTable_;
+
+  public GrpcResultsService(IResultTable                resultTable,
+                            ILogger<GrpcResultsService> logger)
+  {
+    logger_      = logger;
+    resultTable_ = resultTable;
+  }
+
+  public override async Task<GetOwnerTaskIdResponse> GetOwnerTaskId(GetOwnerTaskIdRequest request,
```

Comment:
I think the API should be `getResult` directly, which returns all the info about the result, not just its `ownerTaskId`.

Reply:
This was to reduce the data in transit

Reply:
A large a message would be? I believe less than a kilobyte, so less than a tcp packet. This is fine.

Reply:
It returns multiple values

Reply:
my bad.